first_hour_on_vwb/widget_utils.py

```diff
@@ -57,6 +57,26 @@ def list_group_members(json_string):
     html += "</table>"
     return html
 
+def list_workspace_ids(json_string):
+    """ Return a list of workspace IDs.
```

Comment:
In the comment, clarify that the input to this function is the output from terra workspace list

Reply:
done!